src/components/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  createTodo: (data: Todo) => void;
+  handleError: (value: string) => void;
+}
+
+export const Header: React.FC<Props> = ({ createTodo, handleError }) => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const HandleSubmit = (title: string) => {
```

Comment:
why you put the value if you already have it?
```suggestion
  const handleSubmit = () => {
```